.github/workflows/build.yml

```
@@ -27,6 +27,6 @@ jobs:
         git config user.email "<>"
         git commit -m "[data] Update spec info" -a || true
     - name: Push changes
-      uses: ad-m/github-push-action@v0.5.0
+      uses: ad-m/github-push-action@master
```

```suggestion
      uses: ad-m/github-push-action@0.6.0
```
it feels risky to automatically adopt upgrades to github actions with access to privileges rights without at least some review